### PR TITLE
Explain a new technique for fixing a GCB failure if it has gone undetected for a while.

### PR DESCRIPTION
**Pull request by ndmckinley**
If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: .ci/README.md [hunk 1]
```diff
@@ -38,20 +38,49 @@ or
...
+
+When a task fails, all the other tasks which are part of the same job are cancelled.  However, if one of those tasks has already succeeded, the next task of the same kind - the next Ansible pusher, say, since Ansible tends to finish quickly - will be free to start.  That means that all subsequent jobs have partially succeeded - they pushed Ansible, but not TPGB.  You'll need to do the work of the failed tasks yourself in order to restore the steady state.
+
+When this happened the first time, Cameron and Nathan wrote this little shell snippet, which should do it for you.  You will need to get the Magician's github token, either by generating a new one (be sure to clean up after yourself when done) or by decrypting the value in .ci/gcb-push-downstream.yml.
```
Comment (c2thorn) [inline .ci/README.md:67]: Is the token also available via valentine? Or was that something else

**Review by rileykarson — APPROVED** [review]

File: .ci/README.md (hunk 1 above)
Comment (ndmckinley) [reply .ci/README.md:67]: No, unfortunately.  I could add my new one, though...

**Review by c2thorn — APPROVED** [review]
I think this sufficiently covers the important bits. Thanks Nathan!

File: .ci/README.md (hunk 1 above)
Comment (ndmckinley) [reply .ci/README.md:67]: I did add it, so let me update this to say you can fetch it from there...

**Pull request merged**